### DISCUSSION TITLE
comfyui right-click menu font is too small

### DISCUSSION THREAD

**Discussion by syd2023** [Comfy-Org/ComfyUI#3029] [Q&A]
How do I turn it up

**Comment by Bartabok** [comment]
I got the same problem !! I don't know what i did... a shortcut by mistake ? 
I'm not sure if it's a problem with COmfyUI or with Firefox i use (probably not Firefox as i got this problem only in comfyUI)... but now my right click menus are reallu tiny, near to impossible to read. 
Same for the menu on the right, the one with the "Queue Prompt" button and Manager.

Plz help !

**Reply by Bartabok** [reply]
Found ! :D
It was a firefox problem. I probably used a shortcut without knowing it as my firefox Zoom was set as 50%.
The strange thing is i knew the Ctrl+mouse wheel way to change the web navigator zoom, but it didn't change anything in this case.

anyway... my problem is solve. Thx !
